spotify_visualiser.py

```diff
@@ -28,6 +28,18 @@
     df["master_metadata_track_name"] = df["master_metadata_track_name"] + " | " + df["master_metadata_album_artist_name"]
     st.dataframe(df.head())
 
+    st.write("### Overall Stats :bar_chart:")
+    top_artist = df["master_metadata_album_artist_name"].mode()[0]
+    top_album = df["master_metadata_album_album_name"].mode()[0]
+    top_track = df["master_metadata_track_name"].mode()[0]
+
+    overall_stats = {
+    "Category": ["Top Artist", "Top Album", "Top Track"],
```

Comment:
indents please between the curly brackets (flake8 is complaining)

Reply:
Also I'm doing the PR through VS code using the github extension, I think it's worth getting.